action.yml

```diff
@@ -78,6 +78,6 @@ outputs:
   cache-hit:
     description: 'A boolean value to indicate an exact match was found for the primary key'
 runs:
-  using: 'node16'
+  using: 'node21'
```

Comment:
Could you please change it to node20. If you used the `npm ci` command with node 21 on local machine you need to change it node20 and regenerate dependencies with it.

Reply:
@dmitry-shibanov  Updated node version to 2o and build sucessful in my local.